libs/net/services/Managers/IngestManager`.cs

```diff
@@ -162,6 +162,7 @@ public override async Task RunAsync()
                         // Update ingest with failure.
                         await manager.RecordFailureAsync(ex);
                         this.State.RecordFailure();
+                        await this.SendEmailAsync($"Ingest '{ingest.Name}' failed to run. This is failure {manager.Ingest.FailedAttempts+1} out of {manager.Ingest.RetryLimit} maximum retries.", ex);
```

Comment:
This will send an email for every failure won't it?  We only want to get an email if the failure keeps occurring after retries.

Reply:
Ok. Got it !